Assets/Scripts/Lever.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Lever : MonoBehaviour {
```

Comment:
Add `[RequireComponent(typeof(Animator))]` to make sure that `Animator` component is attached along with the `Lever` script.